docs/README.md

```diff
@@ -37,6 +37,52 @@ See [Protocol Versions](01-Protocol-Versions) for specifics.
 
 See [the PHP library documentation](https://github.com/paragonie/past/blob/master/docs/02-PHP-Library).
 
+## What are PAST's design goals?
+
+### 1. Resistance to Implementation Error / Misuse
+
+While it will be possible for motivated developers to discover novel ways to
+make any tool insecure, PAST attempts to make it easier to develop secure
+implementations than to develop insecure implementations of the standard.
+
+To accomplish this goal, we cast aside runtime protocol negotiation and
+so-called "algorithm agility" in favor of pre-negotiated protocols with
+version identifiers.
+
+For `local` tokens, we encrypt them exclusively using authenticated encryption
+with additional data (AEAD) modes that are also nonce-misuse resistant (NMR).
+This means even if implementations fail to use a secure random number generator
+for the large nonces, message confidentiality isn't imperiled.
+
+### 2. Usability
+
+Developers who are already familiar with JSON Web Tokens (JWT) should be able
+to, intuitively, use PAST in their software with minimal friction.
+
+Additionally, developers who are not already familiar with JWT should be able
+to pick up PAST and use it successfully without introducing security flaws
+into their application.
+
+### 3. Flexibility
+
+Although our library aims to maximize JWT compatibility, out cryptography
```

Comment:
JWT have an equivalent to this, a JWS. To say that PAST can handle more payload types than JOSE seems disingenuous.